source/org/zfin/db/postGmakePostloaddb/1157/db.changelog.master.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+<!--
+    <include file="source/org/zfin/db/postGmakePostloaddb/1157/ZFIN-9186.sql" />
```

Comment:
Is this a placeholder for a future pull request?

Reply:
Yes. It'll be removed with the next PR that contains a LB script.